pub/docs/js.md

```diff
@@ -1,74 +1,251 @@
+
+* _symbol_: ``
+* _pronunciation_:
+* _examples_:
+```javascript
+
+```
+
+### [Operator: Addition](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Addition_(.2B))
+
+* _symbol_: `+`
+* _pronunciation_: "plus", "added to"
+* _examples_:
+```javascript
+1 + 2 + 3 // 6, of course
+'1' + 2 + 3 // '123', obviously... WTF?
+```
+=======
+## Javascript Documentation
+
+### [Primative: Null](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/null)
+
+* _symbol_: `null`
+* _pronunciation_: "null", "null value", "is null"
+* _examples_:
+```javascript
+null // how exciting
+```
+
+### [Primative: Boolean](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean)
+
+* _symbol_: `true` and `false`
+* _pronunciation_: "True" and "False"
+* _examples_:
+```javascript
+'hello'==4 // false
+```
+
+### [Primative: Number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number)
+
+* _symbol_: `new Number(value);`
+* _pronunciation_: "number"
+* _examples_:
+```javascript
+Number("123") // 123
+```
+
+### [Primative: String](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String)
+
+* _symbol_: `"string"`
+* _pronunciation_: "string 'this is a string'"
+* _examples_:
+```javascript 
+"this is a string"
+```
+
+### [Literal: Object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)
+
+* _symbol_: `object(x)`
+* _pronunciation_: "object(stuff)"
+* _examples_:
+```javascript
+var object = {} // creates an empty object with no properties
+```
+
+### [Literal: Array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array)
+
+* _symbol_: `[array contents]`
+* _pronunciation_: array - contents
+* _examples_:
+```javascript
+[element0, element1, ..., elementN]
+```
+
+### [Literal: Regular Expression](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp)
+
+* _symbol_: ``
+* _pronunciation_:
+* _examples_:
+```javascript
+
+```
+
+### [Operator: Addition](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Addition_(.2B))
+
+* _symbol_: `+`
+* _pronunciation_: "plus", "added to"
+* _examples_:
+```javascript
+1 + 2 + 3 // 6, of course
+'1' + 2 + 3 // '123', obviously... WTF?
+```
+
+### [Operator: Modulus](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Arithmetic_operators)
+
+* _symbol_: `%`
+* _pronuntciation_: 'mod u lus'
+* _examples_:
+```javascript
+12 % 5 // 2 and 15 % 3 // 0
+```
+
+### [Operator: Increment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Arithmetic_operators)
+
+* _symbol_: `++`
+* _pronunciation_: `in cre ment`
+* _examples:
+```javascript
+4++ // 5, 6, 7....
+```
```

Comment:
Actually, that won't work, either. Try it!

What _kind_ of operator is the increment?